test/utils.test.ts

```diff
@@ -77,6 +77,12 @@ describe('isRdpcMember', () => {
   });
 });
 
+describe('isRdpcAdmin', () => {
+  it('should return false if failed', () => {
+    expect(validator.isRdpcAdmin(validator.getPermissionsFromToken('ssdfg'))).toBe(false);
+  });
+});
+
```

Comment:
Need updated JWT w/ RDPC Admin permissions for positive test

Reply:
I can generate this for you and send over.

Reply:
```
eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJpYXQiOjE2MjE3ODkzMzgsImV4cCI6MjA1Mjc2NTc2Nywic3ViIjoiZjAzYWMxZGQtY2RmYS00NzRiLTlmYTEtNGMzYzFmNDNhYjBlIiwiaXNzIjoiZWdvIiwiYXVkIjpbXSwianRpIjoiNTg1YzAxYzYtNDUwOS00ZTk3LTg2NWEtMmMwNzZkY2M4OGEzIiwiY29udGV4dCI6eyJzY29wZSI6WyJSRFBDLUNBLldSSVRFIl0sInVzZXIiOnsiZW1haWwiOiJqb2hubnkuYWRtaW5AZXhhbXBsZS5jb20iLCJzdGF0dXMiOiJBUFBST1ZFRCIsImZpcnN0TmFtZSI6IkpvaG5ueSIsImxhc3ROYW1lIjoiUkRQQyBBZG1pbiIsImNyZWF0ZWRBdCI6MTYyMTc4OTMzODAwMCwibGFzdExvZ2luIjoxNjIxOTc2NTM4MDAwLCJwcmVmZXJyZWRMYW5ndWFnZSI6bnVsbCwidHlwZSI6IlVTRVIiLCJwcm92aWRlclR5cGUiOiJHT09HTEUiLCJwcm92aWRlclN1YmplY3RJZCI6ImRhY29BZG1pbjEyMzQifX19.eFna5LLa9K4GdX55iIkmIviK4vU-v4SZGfpBu8po6MOyj7CDi30piqHMGFz0C0dHj6a0Hb7RxOPzEKC2MoMHbvc6A4JwH9IucOb9YW6SD_Qi99Y6QaJoUWMimj-CWzYEgv6V78gFZSN5i1PRoizxfvZRIawTSp_iO2kl6TWpyctQ8P2SIed3riJ6E1Z6uVFp4m3Hzgj3ZAc519vFIu2q2rCnDyb2C-tgzcxGPE6uKnn60MksYYg_qngQe8W5icSbYNjtQhc87V9CwFaZNtmWaKPc59axCapfKcR7hjBBQvSqx5sWq9UZ77oYyAWQYhThcUJSNuUTG18eS8SUBDReFw
```

Reply:
Updated w/ token, added positive/negative test cases